### DISCUSSION TITLE
Does focusOnError support controller component like antd?

### DISCUSSION THREAD

**Discussion by hthuong09** [react-hook-form/react-hook-form#2892] [Q&A] [answered]
Hello, I wonder if focusOnError officially supports controller component like antd?

Forked from yup validator sample.
https://codesandbox.io/s/react-hook-form-validationschema-v6-forked-tcjn6?file=/src/index.js
Replace an uncontrolled input with antd's Input.

Try to leave the age blank and submit.
The error showed up but the field is not focused.

**Comment by bluebill1049** [comment, marked as answer]
yes, we do, but some manual work... because we don't have access to the ref.

https://react-hook-form.com/api#Controller
`onFocus`

https://codesandbox.io/s/react-hook-form-controllerautofocus-v6-eeo66

**Reply by hthuong09** [reply]
It worked! Thanks, @bluebill1049!
